selendroid-server/src/main/java/io/selendroid/server/model/DefaultAppiumUiAutomatorDriver.java

```diff
@@ -0,0 +1,279 @@
+/*
+ * Copyright 2012-2014 eBay Software Foundation and selendroid committers.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.selendroid.server.model;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.math.BigInteger;
+import java.security.SecureRandom;
+import java.util.*;
+import io.selendroid.server.ServerInstrumentation;
+import io.selendroid.server.android.InstrumentedKeySender;
+import io.selendroid.server.android.KeySender;
+import io.selendroid.server.android.internal.Dimension;
+import io.selendroid.server.common.exceptions.NoSuchMethodImplementationException;
+import io.selendroid.server.common.exceptions.SelendroidException;
+import io.selendroid.server.util.SelendroidLogger;
+
+
+public class DefaultAppiumUiAutomatorDriver implements SelendroidDriver {
+
+    public static final String BROWSER_NAME = "browserName";
```

Comment:
why is this public static; will these constants be used out side this context; is there better mechanism for declaring constants